src/main/java/com/exasol/sql/expression/ExpressionTerm.java

```diff
@@ -99,4 +104,43 @@ public static ColumnReference column(final String column) {
     public static ColumnReference column(final String table, final String column) {
         return ColumnReference.column(table, column);
     }
+
+    /**
+     * Create a binary arithmetic expression.
+     * 
+     * @param operator arithmetic operator
+     * @param left left member or the expression
+     * @param right right member of the expression
+     * @return binary arithmetic expression
+     */
+    public static BinaryArithmeticExpression arithmeticExpression(final BinaryArithmeticOperator operator,
+            ValueExpression left, ValueExpression right) {
+        return BinaryArithmeticExpression.of(operator, left, right);
+    }
+
+    /**
+     * Create an Exasol function.
+     *
+     * @param functionName a name of function
+     * @param valueExpressions one or more value expression
+     * @return <code>this</code> instance for fluent programming
+     */
+    public static ExasolFunction exasolFunction(final FunctionName functionName,
```

Comment:
It won't work because I don't send any Exasol-specific data to the constructor. The arguments are just interfaces. If I add another dialect Java won't know which method it should use